<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchedulerGenerator</class>
 <widget class="QMainWindow" name="SchedulerGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1347</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QListWidget" name="listWidget">
      <item>
       <property name="text">
        <string>1 - Enter the scheduler's name</string>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2 - Property</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3 - Specifications</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4 - Restrictions</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5 - Finish</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page1">
       <widget class="QPlainTextEdit" name="nameEdit">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>30</y>
          <width>321</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>33</x>
          <y>40</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Enter the scheduler's name:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="firstOkBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>550</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page2">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>201</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Select the scheduler's type:</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>160</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="staticRdn">
           <property name="text">
            <string>Static</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="dynamicRdn_2">
           <property name="text">
            <string>Dynamic</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="secondOkBtn">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>550</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
       <widget class="QPushButton" name="firstBackBtn">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>550</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page3">
       <widget class="QLabel" name="formulaLabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>501</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="eraseBtn">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>10</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>‚Üê</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>130</y>
          <width>107</width>
          <height>135</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Order</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::TextBrowserInteraction</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="incRdn">
           <property name="text">
            <string>Increasing</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="decRdn">
           <property name="text">
            <string>Decreasing</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="randomRdn">
           <property name="text">
            <string>Random</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="fifoRdn">
           <property name="text">
            <string>FIFO</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>130</y>
          <width>160</width>
          <height>361</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Variables</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cpuCoresBtn">
           <property name="text">
            <string>CPU cores</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cpuProcessingbtn">
           <property name="text">
            <string>CPU Processing Power</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="gpuCoreCBtn">
           <property name="text">
            <string>GPU cores</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="gpuProcesingPowerBtn">
           <property name="text">
            <string>GPU Processing Power</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="taskProcessingSizeBtn">
           <property name="text">
            <string>Task Processing Size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="taskCommSizeBtn">
           <property name="text">
            <string>Task Communication Size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="taskOffloadingBtn">
           <property name="text">
            <string>Task offloading</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RunningMFLOPSbtn">
           <property name="text">
            <string>Running MFLOPS</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="lastCompletionTimeBtn">
           <property name="text">
            <string>Last Completion Time</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="constantBtn">
           <property name="text">
            <string>Constant</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QDoubleSpinBox" name="constBox">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>460</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="maximum">
         <double>1000000000000000000000.000000000000000</double>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>130</y>
          <width>162</width>
          <height>205</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Operators and precedence</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sumBtn">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="minusBtn">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="divisionBtn">
           <property name="text">
            <string>/</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="multiBtn">
           <property name="text">
            <string>*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="openBtn">
           <property name="text">
            <string>(</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="closeBtn">
           <property name="text">
            <string>)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="secondBackBtn">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>550</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QPushButton" name="thirdOkBtn">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>550</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page4">
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>121</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Restrictions</string>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>353</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Restrict</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="restrictionSpinBox">
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>tasks per resource</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="thirdBackbtn">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>550</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QPushButton" name="lastOkBtn">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>550</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="noRestrictBtn">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>170</y>
          <width>96</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>No restriction</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page5">
       <widget class="QLabel" name="confirmLbl">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>160</y>
          <width>471</width>
          <height>201</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="finishBtn">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>550</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Finish</string>
        </property>
       </widget>
       <widget class="QPushButton" name="lastBack">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>550</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
